src/core/actions/index.js

```diff
@@ -21,8 +21,9 @@ import {
   errorToGetUserLogTimeEntries,
   showUserLogTimeEntries
 } from './userLogTimeEntries.action'
+import { isWeb } from '../helpers'
 
-if (typeof window !== 'undefined') {
+if (isWeb()) {
```

Comment:
@GeoDoo the purpose of this function is to detect browser ?

Reply:
@ricfrank Yep!

Reply:
@GeoDoo ok. Maybe is better isBrowser or something like this.